assets/base.css

```diff
@@ -1428,6 +1428,10 @@ button.shopify-payment-button__button--unbranded {
   display: flex;
 }
```

Comment:
- [ ] I might have asked this before. Is the `add to cart` button text supposed to change once you've added the product to the cart ? [video](https://screenshot.click/17-50-fyjwy-xjqjv.mp4)

Reply:
it stays the same.

Reply:
no, it will stay the same.

Reply:
Great, thanks D!